content/cli/v6/using-npm/scripts.md

```diff
@@ -31,7 +31,7 @@ To create "pre" or "post" scripts for any scripts defined in the `"scripts"` sec
 
 ### Life Cycle Scripts
 
-There are some special life cycle scripts that happen only in certain situations. These scripts happen in addtion to the "pre" and "post" script.
+There are some special life cycle scripts that happen only in certain situations. These scripts happen in addition to the "pre" and "post" script.
```

Comment:
This field is sourced from https://github.com/npm/cli in the old v6 branch and is likely going to remain as it is.